docs/architecture/microservices/net-core-net-framework-containers/official-net-docker-images.md

```diff
@@ -10,7 +10,9 @@ The Official .NET Docker images are Docker images created and optimized by Micro
 
 Since .NET Core 2.1, all the .NET Core images, including for ASP.NET Core are available at Docker Hub at the .NET Core image repository: <https://hub.docker.com/_/microsoft-dotnet-core/>.
 
-Most image repos provide extensive tagging to help you select not just a specific framework version, but also to choose an OS (Linux distro or Windows version).
+Since May 2018, Microsoft images are being [syndicated in the Microsoft Container Registry](https://azure.microsoft.com/en-us/blog/microsoft-syndicates-container-catalog/). The official catalog is still only available in Docker Hub, and there you'll find the updated address to pull the image.
```

Comment:
```suggestion
Since May 2018, Microsoft images are being [syndicated in the Microsoft Container Registry](https://azure.microsoft.com/blog/microsoft-syndicates-container-catalog/). The official catalog is still only available in Docker Hub, and there you'll find the updated address to pull the image.
```